client/src/shared/ui/form/TextInput.js

```diff
@@ -95,6 +85,9 @@ export default function TextInput(props) {
         <FormFeedback
           error={ error }
         />
+        {description &&
+          <p className="form-control">{description}</p>
+        }
```

Comment:
```suggestion
        { description &&
          <p className="form-control">{ description }</p>
        }
```

Reply:
There are still some cases in this file that could need some spaces inside brackets. 🙂